src/app/pages/setting/setting.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { Router } from '@angular/router';
+
+
+@Component({
+  selector: 'app-setting',
+  templateUrl: './setting.component.html',
+  styleUrls: ['./setting.component.css']
+})
+export class SettingComponent implements OnInit {
+
+  public check1: boolean = true;
```

Comment:
why don't you use the right variable name for checkboxes? what is check1?? How would one understand your code when he/she read it by just saying check1?